crypto/rsa_common.go

```diff
@@ -1,10 +1,12 @@
 package crypto
 
 import (
-	"errors"
+	"fmt"
 )
 
+const MinRsaKeyBits = 2048
```

Comment:
I wonder if the go runtime exposes a flag/property/env variable we can inspect to know if we're running inside a test, in which case we could turn this into a `var` and close over it:

```go
var MinRsaKeyBits = func() int {
  ret := 2048
  if runtime.IsTest() {
    ret = 512
  }
  return ret
}()
```

Reply:
There isn't a simple one and IMO this isn't a clean solution.

Reply:
Note: This is also a issue for integration (non go) tests. However, we can probably set an environment variable there (or, again, consider just not using RSA for most tests).